test/Sign.Core.Test/Tools/VSIXSignTool/Crypt32Tests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+using Sign.Core.Interop;
+using System.Runtime.InteropServices;
+
+namespace Sign.Core.Test
+{
+    public class Crypt32Tests
+    {
+        [Fact]
+        public void ShouldTimestampData()
+        {
+            var data = new byte[] { 1, 2, 3 };
+            var parameters = new CRYPT_TIMESTAMP_PARA
+            {
+                cExtension = 0,
+                fRequestCerts = true,
+                pszTSAPolicyId = null
+            };
+
+            var ok = Crypt32.CryptRetrieveTimeStamp("http://timestamp.digicert.com", CryptRetrieveTimeStampRetrievalFlags.NONE, 30 * 1000, "1.3.14.3.2.26", ref parameters, data, (uint)data.Length, out var pointer, IntPtr.Zero, IntPtr.Zero);
```

Comment:
Tests should use a [local, test timestamp server](https://github.com/dotnet/sign/blob/fc6b4d230543921dd72f7f7063814e68457fd262/test/Sign.Core.Test/TestInfrastructure/Server/CertificatesFixture.cs#L20).

You'll probably also need to update from SHA-1 (`1.3.14.3.2.26`) to SHA-2.